image/index.css

```diff
@@ -0,0 +1,373 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.bigContainer {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
+  background-color: #f0f8ff;
+}
+
+.navbar_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  gap: 3%;
+  align-items: center;
+  height: 180px;
+  width: 100%;
+  background-color: #729762;
+  border: 1px solid #729762;
+}
+.img {
```

Comment:
Give a proper name for according to the usage of the tag or purpose
Like if using for footer image `img` -> `footer_image`